iotedgedev/modules.py

```diff
@@ -66,72 +65,71 @@ def push(self, no_build=False):
         self.build_push(no_build=no_build)
 
     def build_push(self, no_build=False, no_push=False):
-
         self.output.header("BUILDING MODULES", suppress=no_build)
 
-        # Get all the modules to build as specified in config.
-        modules_to_process = self.utility.get_active_modules()
-
-        for module in os.listdir(self.envvars.MODULES_PATH):
-
-            if len(modules_to_process) == 0 or modules_to_process[0] == "*" or module in modules_to_process:
-
-                module_dir = os.path.join(self.envvars.MODULES_PATH, module)
-
-                self.output.info("BUILDING MODULE: {0}".format(module_dir), suppress=no_build)
-
-                module_json = Module(self.output, self.utility, os.path.join(module_dir, "module.json"))
-                mod_proc = ModulesProcessorFactory(self.envvars, self.utility, self.output, module_dir).get(module_json.language)
+        deployment_manifest = DeploymentManifest(self.envvars, self.output, self.envvars.DEPLOYMENT_CONFIG_TEMPLATE_FILE, True)
```

Comment:
Breaking changes here: now it will parse deployment.template.json to find modules to process instead of respecting the `ACTIVE_MODULES` env. @jongio Please tell me if you have any concern or suggestion.